thorlcr/activities/choosesets/thchoosesetsslave.cpp

```diff
@@ -30,7 +30,7 @@ class BaseChooseSetsActivity : public CSlaveActivity
     unsigned *tallies;
 
 public:
-    BaseChooseSetsActivity(CGraphElementBase *_container) : CSlaveActivity(_container)
+    BaseChooseSetsActivity(CGraphElementBase *_container, const StatisticsMapping & _mapping = basicActivityStatistics) : CSlaveActivity(_container, _mapping)
```

Comment:
Clearer/safer to not have a default parameter so it is easy to check the correct mapping is being used for all variants.